app/src/test/java/com/saltedge/authenticator/features/authorizations/details/AuthorizationDetailsPresenterTest.kt

```diff
@@ -62,74 +59,42 @@ class AuthorizationDetailsPresenterTest {
 
     @Test
     @Throws(Exception::class)
-    fun remainedTimeDescriptionTest() {
+    fun startTimeTest() {
         val presenter = createPresenter(viewContract = mockView)
-        presenter.setInitialData(
-            connectionId = "1", authorizationId = "",
-            viewModel = null, quickConfirmMode = false
-        )
+        presenter.currentViewModel = null
 
-        assertThat(presenter.remainedTimeDescription, equalTo(""))
+        Assert.assertNull(presenter.startTime)
 
-        val authorizationData = createAuthorizationData(1)
-        presenter.setInitialData(
-            connectionId = "1",
-            authorizationId = "",
-            viewModel = authorizationData.toAuthorizationViewModel(connection1),
-            quickConfirmMode = false
-        )
+        val authorization = viewModel1
```

Comment:
Do we need `authorization` variable?

Maybe we should reconsider the name of the `viewModel1` variable?